core/trino-main/src/main/java/io/trino/SystemSessionProperties.java

```diff
@@ -75,7 +75,7 @@
     public static final String PUSH_TABLE_WRITE_THROUGH_UNION = "push_table_write_through_union";
     public static final String EXECUTION_POLICY = "execution_policy";
     public static final String DICTIONARY_AGGREGATION = "dictionary_aggregation";
-    public static final String PLAN_WITH_TABLE_NODE_PARTITIONING = "plan_with_table_node_partitioning";
+    public static final String USE_TABLE_SCAN_NODE_PARTITIONING = "use_table_scan_node_partitioning";
```

Comment:
Is this backward incompatible change needed? How long was the property around?

Reply:
> Is this backward incompatible change needed? How long was the property around?

I don't think it was used:
1. it was not backed by feature config (I added that recently)
and
2. it was not checked in all relevant places.